VideoApp/VideoApp/Video/LocalMediaController.m

```diff
@@ -162,4 +162,16 @@ - (void)videoCaptureStarted {
     }
 }
 
+- (void)cameraSourceWasInterrupted {
+    if (self.camera.localVideoTrack) {
+        [self.localParticipant unpublishVideoTrack:self.camera.localVideoTrack];
+    }
+}
+
+- (void)cameraSourceInterruptionEnded {
+    if (self.camera.localVideoTrack) {
+        [self.localParticipant publishVideoTrack:self.camera.localVideoTrack];
+    }
+}
```

Comment:
I was trying to be surgical because this is objc. I didn't blow away `localVideoTrack` because the UI uses it to determine state of record video toggle button. But I'm curious if in the future we decouple more code from UI are there any big considerations to keeping local video track around or not that are not obvious?

I confirmed these are called on main thread.

Reply:
This seems to work great for the other participant. It turns video on/off just like we want.

For the user that returns app to foreground they see their previous video for less than a seconds until it updates. This seems to be ideal UX.